src/ripple/consensus/LedgerTrie.h

```diff
@@ -513,58 +529,51 @@ class LedgerTrie
     bool
     remove(Ledger const& ledger, std::uint32_t count = 1)
```

Comment:
Is calling this function with `count == 0` allowed?

Reply:
It should work but not change anything in the trie.